clients/src/main/java/org/apache/kafka/clients/consumer/internals/events/ShareAcknowledgeCommitCallbackHandlerEvent.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.clients.consumer.internals.events;
+
+public class ShareAcknowledgeCommitCallbackHandlerEvent extends ApplicationEvent {
```

Comment:
The callback class is `AcknowledgementCommitCallback` so I think this should be `ShareAcknowledgementCommitCallbackRegistrationEvent`, and the type should then be `SHARE_ACKNOWLEDGEMENT_COMMIT_CALLBACK_REGISTRATION`.

Reply:
Okay yeah makes sense. I have updated the PR.